core/src/main/java/org/mule/runtime/core/privileged/exception/TemplateOnErrorHandler.java

```diff
@@ -437,9 +439,23 @@ public boolean accept(CoreEvent event) {
     return acceptsAll() || (acceptsErrorType(event) && acceptsExpression(event));
   }
 
-
   private boolean acceptsErrorType(CoreEvent event) {
-    return errorTypeMatcher == null || errorTypeMatcher.match(event.getError().get().getErrorType());
+    Error error = event.getError().get();
+    return errorTypeMatcher == null || errorTypeMatcher.match(error.getErrorType())
+        || acceptsSuppressedErrorType(error.getSuppressedErrors(), event);
+  }
+
+  private boolean acceptsSuppressedErrorType(List<Error> suppressedErrors, CoreEvent coreEvent) {
+    for (Error suppressedError : suppressedErrors) {
+      ErrorType suppressedErrorType = suppressedError.getErrorType();
+      if (errorTypeMatcher.match(suppressedErrorType)) {
+        logger
+            .warn("{} error handler has matched the following underlying error: {}. Consider a configuration change in order to match the reported error: {}",
```

Comment:
`Consider a configuration change in order to match the reported error: ` -> `Consider changing it to match the reported error: `